File: pTk/tkInt_f.m
#ifndef _TKINT_F_VM
#define _TKINT_F_VM
#include "tkInt_f_f.h"
#define TkintVptr (*Tkint_fVptr->V_TkintVptr)
#ifndef TkintVGet
#define TkintVGet (*Tkint_fVptr->V_TkintVGet)
#endif
#endif /* _TKINT_F_VM */

File: pTk/tkInt_f.c
#include "tkInt.h"
#include "tkInt_f.h"
static TkintVtab TkintVtable =
{
#define VFUNC(type,name,mem,args) name,
#define VVAR(type,name,mem)      &name,
#include "tkInt.t"
#undef VFUNC
#undef VVAR
};
TkintVtab *TkintVptr;
TkintVtab *TkintVGet() { return TkintVptr = &TkintVtable;}
